sysl2/sysl/seqs/visitor.go

```diff
@@ -207,17 +232,28 @@ func (v *SequenceDiagramVisitor) Visit(e Element) error {
 			log.Errorln(err)
 		}
 	}()
-
+	var err error
 	switch t := e.(type) {
 	case *EndpointCollectionElement:
-		return v.visitEndpointCollection(t)
+		err = v.visitEndpointCollection(t)
+		for bbKey, bbVal := range t.blackboxes {
+			if bbVal.valueType >= BbApplication && bbVal.visitCount == 0 {
```

Comment:
`bbVal.valueType >= BbApplication` is fragile. 
Either add test case or switch on valueType.